apps/retrospective/serializers/retrospective.go

```diff
@@ -23,6 +23,43 @@ type Retrospective struct {
 	StoryPointPerWeek  float64
 }
 
+// EditLevel ...
+type EditLevel uint
+
+// different EditLevels ...
+const (
+	NotEditable EditLevel = iota + 1
+	Partially
+	Fully
+)
+
+// RetroFieldsEditLevelMap ...
+type RetroFieldsEditLevelMap struct {
+	Title             EditLevel
+	ProjectName       EditLevel
+	TeamID            EditLevel
+	StoryPointPerWeek EditLevel
+	TimeProviderName  EditLevel
+}
+
+func (editLevel EditLevel) validate() bool {
+	if editLevel == NotEditable || editLevel == Partially || editLevel == Fully {
```

Comment:
Can be simplified to :
```suggestion
	return (editLevel == NotEditable || editLevel == Partially || editLevel == Fully)
```

Also, we should check if the value is in range so that future additions/removal of the levels doesn't require changes in the logic.

Reply:
Done.